### PR TITLE
Added Traditional Chinese (zh_tw.json)

### PR DESCRIPTION
**Pull request by yichifauzi**
Added Traditional Chinese translation

### CODE REVIEW

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.shop.price_info": "價格：%s",
+  "gui.universal_shops.shop.stock_info": "庫存：%s",
+  "gui.universal_shops.shop.currency_storage": "貨幣儲存",
+  "gui.universal_shops.shop.settings": "商店設定",
+  "gui.universal_shops.shop.title": "%s 的商店",
+  "gui.universal_shops.shop.title.admin": "管理商店",
+
+  "gui.universal_shops.click_to_buy": "點擊物品購買",
+  "gui.universal_shops.out_of_stock": "缺貨！",
+  "gui.universal_shops.setup.cant_change_pricehandler.1": "無法更改此設定！",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:17]:
```suggestion
  "gui.universal_shops.setup.cant_change_pricehandler.1": "無法變更這項設定！",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.max_stock_left": "您可以購買：%s",
+
+  "gui.universal_shops.shop.price_info": "價格：%s",
+  "gui.universal_shops.shop.stock_info": "庫存：%s",
+  "gui.universal_shops.shop.currency_storage": "貨幣儲存",
+  "gui.universal_shops.shop.settings": "商店設定",
+  "gui.universal_shops.shop.title": "%s 的商店",
+  "gui.universal_shops.shop.title.admin": "管理商店",
+
+  "gui.universal_shops.click_to_buy": "點擊物品購買",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:15]:
```suggestion
  "gui.universal_shops.click_to_buy": "按物品以購買",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.out_of_stock": "缺貨！",
+  "gui.universal_shops.setup.cant_change_pricehandler.1": "無法更改此設定！",
+  "gui.universal_shops.setup.cant_change_pricehandler.2": "請確保你已提取所有儲存的貨幣！",
+  "gui.universal_shops.setup.click_to_change_mode.1": "點擊以更改模式",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:19]:
```suggestion
  "gui.universal_shops.setup.click_to_change_mode.1": "按這裡變更模式",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.setup.click_to_change_mode.2": "按住 Shift 鍵向後更改",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:20]:
```suggestion
  "gui.universal_shops.setup.click_to_change_mode.2": "按住 Shift 鍵向後變更",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.setup.stock_type": "庫存類型：%s",
+  "gui.universal_shops.setup.hologram": "全息圖：%s",
+  "gui.universal_shops.setup.allow_hoppers": "允許漏斗：%s",
+  "gui.universal_shops.setup.virtual_balance.currency": "貨幣：%s",
+  "gui.universal_shops.setup.virtual_balance.value": "價格：%s",
+  "gui.universal_shops.modifying_item.title": "修改物品",
+  "gui.universal_shops.modifying_item.stack_size": "設定數量：%s",
+  "gui.universal_shops.modifying_item.stack_decrease": "減少物品數量",
+  "gui.universal_shops.modifying_item.stack_increase": "增加物品數量",
+  "gui.universal_shops.modifying_item.change_item": "更改基本物品",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:31]:
```suggestion
  "gui.universal_shops.modifying_item.change_item": "變更基本物品",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 6]
```diff
@@ -0,0 +1,73 @@
...
+
+  "text.universal_shops.prefix": "商店",
+  "text.universal_shops.not_enough_shop_storage_space": "商店空間不足！",
+  "text.universal_shops.not_enough_currency": "貨幣不足！",
+  "text.universal_shops.not_enough_inventory_space": "背包空間不足！",
+  "text.universal_shops.not_enough_stack_space": "你的游標已滿！",
+  "text.universal_shops.not_enough_stock": "商店庫存不足！",
+  "text.universal_shops.stock_missing": "缺少庫存！",
+  "text.universal_shops.not_set": "未設定！",
+  "text.universal_shops.requires_setup_by_owner": "此商店尚未設定！",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:49]:
```suggestion
  "text.universal_shops.requires_setup_by_owner": "這個商店尚未設定！",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+  "pricehandler.universal_shops.free": "免費",
+  "pricehandler.universal_shops.items": "物品",
+  "pricehandler.universal_shops.virtual_balance": "餘額",
+
+  "stockhandler.universal_shops.single_item": "相同物品",
+  "stockhandler.universal_shops.selected_item": "任何堆疊",
+
+  "hologram_type.universal_shops.full": "完整",
+  "hologram_type.universal_shops.icon": "僅圖示",
+  "hologram_type.universal_shops.disabled": "停用",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:67]:
```suggestion
  "hologram_type.universal_shops.disabled": "已停用",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.modifying_item.stack_decrease": "減少物品數量",
+  "gui.universal_shops.modifying_item.stack_increase": "增加物品數量",
+  "gui.universal_shops.modifying_item.change_item": "更改基本物品",
+  "gui.universal_shops.modifying_item.change_item.title": "選擇物品",
+  "gui.universal_shops.virtual_balance.collect": "領錢",
+  "gui.universal_shops.virtual_balance.stored": "已儲存：%s",
+  "gui.universal_shops.virtual_balance.cost.title": "更改價格",
+  "gui.universal_shops.virtual_balance.cost.invalid_input": "輸入無效！",
+  "gui.universal_shops.virtual_balance.settings.title": "貨幣設定",
+  "gui.universal_shops.virtual_balance.selected_account": "已選帳戶：%s",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:38]:
```suggestion
  "gui.universal_shops.virtual_balance.selected_account": "已選擇帳戶：%s",
```

File: src/main/resources/data/universal_shops/lang/zh_tw.json [hunk 9]
```diff
@@ -0,0 +1,73 @@
...
+  "gui.universal_shops.setup.virtual_balance.value": "價格：%s",
+  "gui.universal_shops.modifying_item.title": "修改物品",
+  "gui.universal_shops.modifying_item.stack_size": "設定數量：%s",
+  "gui.universal_shops.modifying_item.stack_decrease": "減少物品數量",
+  "gui.universal_shops.modifying_item.stack_increase": "增加物品數量",
+  "gui.universal_shops.modifying_item.change_item": "更改基本物品",
+  "gui.universal_shops.modifying_item.change_item.title": "選擇物品",
+  "gui.universal_shops.virtual_balance.collect": "領錢",
+  "gui.universal_shops.virtual_balance.stored": "已儲存：%s",
+  "gui.universal_shops.virtual_balance.cost.title": "更改價格",
```
Comment (notlin4) [inline src/main/resources/data/universal_shops/lang/zh_tw.json:35]:
```suggestion
  "gui.universal_shops.virtual_balance.cost.title": "變更價格",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by notlin4 — APPROVED** [review]

**Comment by Patbox** [comment]
Hey! Translations should be sent via https://crowdin.com/project/patbox-mods as it makes things simple. Sorry for not responding to about this!

**Comment by yichifauzi** [comment]
> Hey! Translations should be sent via https://crowdin.com/project/patbox-mods as it makes things simple. Sorry for not responding to about this!

Already submitted please check it Patbox thanks you for making this mod

**Pull request closed without merging**